.github/CONTRIBUTING.md

```diff
@@ -125,8 +125,8 @@ $ sudo -u postgres psql -c "CREATE EXTENSION pg_trgm;" peertube_dev
 $ sudo -u postgres psql -c "CREATE EXTENSION unaccent;" peertube_dev
 ```
 
-Peertube also requires a running redis server, no special setup is needed for
-this.
+PeerTube also requires a running redis server, no special setup is needed for
+this (it's `sudo service redis-server start` on WSL 1 with Ubuntu).
```

Comment:
Please remove the `(it's `sudo service redis-server start` on WSL 1 with Ubuntu)` as it's not the proper place to have detailed explanations depending on the linux distribution